ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,592 @@
+            roundedCornerView.trailingAnchor.constraint(equalTo: bottomBarView.trailingAnchor),
+            roundedCornerView.topAnchor.constraint(equalTo: bottomBarView.topAnchor),
+            roundedCornerView.bottomAnchor.constraint(equalTo: bottomBarView.bottomAnchor),
+            contentView.leadingAnchor.constraint(equalTo: bottomBarView.leadingAnchor, constant: Constants.BottomBar.heroStackLeadingTrailingMargin),
+            contentView.trailingAnchor.constraint(equalTo: bottomBarView.trailingAnchor, constant: -Constants.BottomBar.heroStackLeadingTrailingMargin),
+            contentView.topAnchor.constraint(equalTo: bottomBarView.topAnchor, constant: Constants.BottomBar.heroStackTopBottomMargin),
+            contentView.bottomAnchor.constraint(equalTo: bottomBarView.bottomAnchor, constant: -Constants.BottomBar.heroStackTopBottomMargin)
+        ])
+
+        return bottomBarView
+    }
+
+    private func makeBottomSheetContent(headerView: UIView, expandedContentView: UIView) -> UIView {
+        let view = UIView()
+        let separator = Separator()
+        separator.translatesAutoresizingMaskIntoConstraints = false
+        headerView.translatesAutoresizingMaskIntoConstraints = false
+        expandedContentView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(headerView)
+        view.addSubview(expandedContentView)
+        view.addSubview(separator)
+        NSLayoutConstraint.activate([
+            headerView.topAnchor.constraint(equalTo: view.topAnchor),
+            headerView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            headerView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            expandedContentView.topAnchor.constraint(equalTo: headerView.bottomAnchor),
+            expandedContentView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            expandedContentView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            expandedContentView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            separator.topAnchor.constraint(equalTo: expandedContentView.topAnchor),
+            separator.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            separator.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        return view
+    }
+
+    private func updateExpandability() {
+        if isInSheetMode {
+            bottomSheetController?.collapsedContentHeight = bottomSheetHeroStackHeight
+            bottomSheetController?.isExpandable = isExpandable
+            bottomSheetHeroStackTopConstraint?.constant = bottomSheetHeroStackTopMargin
+        } else {
+            moreButtonView.isHidden = !isExpandable
+        }
+    }
+
+    private lazy var moreButtonView: UIView = {
+        let moreButtonItem = TabBarItem(title: Constants.BottomBar.moreButtonTitle, image: Constants.BottomBar.moreButtonIcon ?? UIImage())
```

Comment:
What happens if we actually fall through and set an empty `UIImage()` as the image?

Reply:
It would just show the title, but the fluent staticImage API will crash the app even before that, which is probably a good thing.